docs/preview/features/powershell/arm.md

```diff
@@ -56,3 +57,34 @@ PS> Inject-ArmContent -Path deploy\arm-template.json
 
 **Recommendations**
 Always inject the content in your ARM template as soon as possible, preferably during release build that creates the artifact
+
+## Removing resource locks on Azure
+
+In some deployments resource-locks are being assigned. To help in removing these quickly, we have provided you with a function that removes all the existing locks on the resource group.
+
+While this seems dangerous, only users with sufficient access rights are allowed to delete locks.
```

Comment:
Perhaps include some information regarding the specific access rights required:
```suggestion
While this seems dangerous, only users with access to the following actions are allowed to delete locks:
 - _Microsoft.Authorization/*_
 - _Microsoft.Authorization/locks/*_
 
 Of the built-in roles, only **Owner** and **Access Administrator** are granted those actions.
```

Reply:
Ok, thank you for this!